src/main/java/nl/hsac/fitnesse/fixture/util/selenium/SeleniumHelper.java

```diff
@@ -1148,6 +1149,13 @@ public void setWindowSizeToMaximum() {
         getWindow().maximize();
     }
 
+    public void setBrowserPositionXY(int xPos, int yPos) {
+		String currentWindowHandle = driver().getWindowHandle();
+		getTargetLocator().window(currentWindowHandle);
```

Comment:
Are these first two lines needed? And if so do we reset the window to the previous value after changing the position?